tools/serverpod_cli/lib/src/generator/types.dart

```diff
@@ -165,6 +186,8 @@ class TypeDefinition {
 
   bool get isColumnSerializable => columnType == 'ColumnSerializable';
 
+  bool get isSerializableDartType => _whiteListedTypes.contains(className);
```

Comment:
Question: Couldn't the `_whiteListedTypes` be completely replaced by the following?

```suggestion
  bool get isSerializableDartType => ![
    ValueType.record,
    ValueType.isEnum,
    ValueType.classType,
  ].contains(valueType);
```

So we can remove the duplicated list of class names.

Reply:
That actually works. Thanks for pointing it out again. I missed `valueType`